src/tools/ceph-dencoder/CMakeLists.txt

```diff
@@ -45,7 +45,8 @@ if(WITH_RBD)
     cls_rbd_client
     rbd_mirror_types
     rbd_types
-    rbd_replay_types)
+    rbd_replay_types
+    rbd_internal)
```

Comment:
Nit: types should be in `rbd_types` so no need to pull in `rbd_internal`

Reply:
That just means that `cache/pwl/Types.cc` should be moved to the `rbd_types` static library

Reply:
Done